msteams-platform/get-started/prerequisites.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Get started - Prerequisites
+author: adrianhall
+description: Learn how to get started with Microsoft Teams app development and set up your environment.
+ms.author: adhal
+ms.date: 05/18/2021
+ms.topic: quickstart
+---
+# Prerequisites: Get started with Microsoft Teams app development
+
+Before your create your first Teams app, you must install a few tools and set up your development environment.
+
+## Install required tools
+
+Some of the tools you need depend on how you you prefer to build your Teams app:
+
+- [Node.js](https://nodejs.org/en/download/) (use the latest v14 LTS release) 
```

Comment:
For node installation in Linux, we may need to remind user to **RESTART ALL VS Code instances** to make the node detectable by our toolkit, otherwise the dependency checker will still fail.